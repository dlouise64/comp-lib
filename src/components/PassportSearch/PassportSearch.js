import React from 'react';
// import PropTypes from 'prop-types';
import PassportInput from '../PassportInput';
import Separator from '../Separator';
// import Well from '../Well';
import './PassportSearch.css';

class PassportSearch extends React.Component {
  render() {
    return (
      <div className="well">

          <PassportInput />
          <Separator />

      </div>
    );
  }
}

PassportSearch.propTypes = {
};

PassportSearch.defaultProps = {
}

export default PassportSearch;
