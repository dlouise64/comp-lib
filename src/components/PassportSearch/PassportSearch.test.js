import React from 'react';
import renderer from 'react-test-renderer';
import PassportSearch from './PassportSearch';

test('<PassportSearch /> renders correctly', () => {
  const tree = renderer.create(<PassportSearch />).toJSON();
  expect(tree).toMatchSnapshot();
});
