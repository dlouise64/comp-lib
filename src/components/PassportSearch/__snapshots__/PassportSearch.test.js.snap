// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PassportSearch /> renders correctly 1`] = `
<div
  className="well"
>
  <div
    className="PassportInput"
  >
    <div
      style={
        Object {
          "display": "inline-block",
          "marginBottom": 16,
          "position": "relative",
        }
      }
    >
      <label
        className="label"
        htmlFor="registration-form-passport_no"
        style={
          Object {
            "display": "inline-block",
            "marginRight": "10px",
          }
        }
      >
        Passport no.
         
        <span
          style={
            Object {
              "color": "red",
            }
          }
        >
           *
        </span>
      </label>
      <input
        className="textInput"
        id="registration-form-passport_no"
        name="passport_no"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <input
      className="btn info"
      onClick={[Function]}
      style={
        Object {
          "marginLeft": "10px",
        }
      }
      type="submit"
      value="Submit"
    />
  </div>
  <div
    className="separator"
  />
</div>
`;
